StackCalculator/StackCalculator/IStack.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace StackCalculator;
+
+/// <summary>
+/// Represents a first-in-last-out collection of floats.
+/// </summary>
+public interface IStack
+{
+    /// <summary>
+    /// Gets a value indicating whether the stack is empty.
+    /// </summary>
+    bool IsEmpty { get; }
+
+    /// <summary>
+    /// Returns last element and erases it.
+    /// </summary>
+    /// <returns>Last element in the stack and bool flag indicating whether stack was not empty.</returns>
+    (float element, bool isPopped) Pop();
```

Comment:
Лучше явно указать, пока разрешены не только public
```suggestion
    public (float Element, bool IsPopped) Pop();
```